### PR TITLE
Drop the 2021-03-19 pipeline; move 2021-04-19 to prod

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: pipeline/terraform/stack/cluster.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ resource "aws_ecs_cluster" "cluster" {
 }
 
 module "inference_capacity_provider" {
-  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/ec2_capacity_provider?ref=v3.4.0"
+  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/ec2_capacity_provider?ref=v3.5.1"
```
Comment (jamesgorrie) [inline pipeline/terraform/stack/cluster.tf:7]: `1` not `2`?

**Review by jamesgorrie — APPROVED** [review]
What did we gain with the `4=>5` upgrade.

File: pipeline/terraform/stack/cluster.tf (hunk 1 above)
Comment (alexwlchan) [reply pipeline/terraform/stack/cluster.tf:7]: Oops, although not an issue for this module.

**Comment by alexwlchan** [comment]
> What did we gain with the 4=>5 upgrade.

Terraform now manages the CloudWatch log groups for the log router (https://github.com/wellcomecollection/terraform-aws-ecs-service/pull/33), so they'll be cleaned up when we tear down a pipeline.

**Pull request merged**